lib/views/after_auth_screens/events/event_calendar.dart

```diff
@@ -156,6 +154,21 @@ _AppointmentDataSource _getCalendarDataSource(List<Event> eventsList) {
   return _AppointmentDataSource(appointments);
 }
 
+/// a_line_ending_with_end_punctuation.
+///
+/// **params**:
+/// * `time`: define_the_param
+///
+/// **returns**:
+/// * `DateTime`: define_the_return
+DateTime _parseTime(String time) {
+  try {
+    return DateFormat('h:mm a', 'en_US').parse(time);
+  } catch (e) {
+    return DateFormat('Hms', 'en_US').parse(time);
+  }
```

Comment:
Specify type of error using on clause, on return 'Hms' in case format exception, otherwise throw an error.
its not big issue, but for better code quality.